src/verify_ce_config.py

```diff
@@ -0,0 +1,64 @@
+#!/bin/env python
+
+"""
+Verify HTCondor-CE configuration before service startup
+"""
+
+import re
+import sys
+
+# Verify that the HTCondor Python bindings are in the PYTHONPATH
+try:
+    import classad
+    import htcondor
+except ImportError:
+    sys.exit("ERROR: Could not load HTCondor Python bindings. " + \
+             "Please ensure that the 'htcondor' and 'classad' are in your PYTHONPATH")
+
+# Create dict whose values are lists of ads specified in the relevant JOB_ROUTER_* variables
+JOB_ROUTER_CONFIG = dict((x, [x for x in classad.parseAds(htcondor.param[x])])
+                         for x in ['JOB_ROUTER_DEFAULTS', 'JOB_ROUTER_ENTRIES'])
```

Comment:
Well, if I was picky, you get more carried away later with if's inside the comprehension.